loom/engine/bindings/loom/lmApplicationTasks.cpp

```diff
@@ -33,8 +33,6 @@ extern "C"
 
 void loom_tick()
 {
-    LOOM_PROFILE_START(loom_tick);
```

Comment:
Not really in favor of, but I don't think it gets tracked accurately anyway, since the block itself contains the profiling tick and when the profile gets dumped, the tick is in progress and therefore fails the zero stack check (we could add an exception, but that's kind-of odd).